<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IntLoopShaping</class>
 <widget class="QDialog" name="IntLoopShaping">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>486</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Selecciona el algoritmo a usar:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Introduce Epsilon:</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>167</width>
     <height>108</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="sachin">
      <property name="text">
       <string>A&amp;lgoritmo Sachin</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="nand">
      <property name="text">
       <string>Algoritmo &amp;Nandkishor</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="isaac">
      <property name="text">
       <string>Algorit&amp;mo Isaac</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="ram">
      <property name="text">
       <string>Al&amp;goritmo Rambabú</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>320</y>
     <width>158</width>
     <height>34</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Epsilon:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="epsilon"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>400</y>
     <width>86</width>
     <height>76</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QPushButton" name="cancel">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ok">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>40</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Rango de representación:</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>60</y>
     <width>193</width>
     <height>116</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Inicio:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="inicio"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Final:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="final_2"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>N Puntos:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="nPuntos"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QRadioButton" name="depuracion">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>97</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Depuraci&amp;ón</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>92</width>
     <height>54</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QRadioButton" name="linspace">
      <property name="text">
       <string>LinSpace</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="logspace">
      <property name="text">
       <string>LogSpace</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QRadioButton" name="hilos">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>380</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>E&amp;jecutar por hilos</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="datosAlg">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>199</y>
     <width>231</width>
     <height>181</height>
    </rect>
   </property>
   <widget class="QWidget" name="vacia"/>
   <widget class="QWidget" name="page_isaac">
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>71</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Opciones:</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>158</width>
       <height>80</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QCheckBox" name="bisectionAvanced">
        <property name="text">
         <string>Bisección Avanzada</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="aceleratedAvanced">
        <property name="text">
         <string>Acelerated Avanzada</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="deteccionAvanced">
        <property name="text">
         <string>Detección Avanzada</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="page_nand">
    <widget class="QWidget" name="layoutWidget6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>111</width>
       <height>83</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QRadioButton" name="centro">
        <property name="text">
         <string>Cen&amp;tro</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="aleatorio">
        <property name="text">
         <string>Aleatorio</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="superior">
        <property name="text">
         <string>Superior bo&amp;x</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="inicializacion">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Tipo inicialización:</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>111</width>
       <height>34</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_delta">
        <property name="text">
         <string>Delta:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="delta"/>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
